static/src/js/schemas/uiSchemas/collections/temporalInformation.js

```diff
@@ -0,0 +1,376 @@
+                        {
+                          'ui:col': {
+                            md: 12,
+                            children: ['EndDate']
+                          }
+                        }
+                      ]
+                    },
+                    {
+                      'ui:row': [
+                        {
+                          'ui:col': {
+                            md: 12,
+                            children: ['DurationUnit']
+                          }
+                        }
+                      ]
+                    },
+                    {
+                      'ui:row': [
+                        {
+                          'ui:col': {
+                            md: 12,
+                            children: ['DurationValue']
+                          }
+                        }
+                      ]
+                    },
+                    {
+                      'ui:row': [
+                        {
+                          'ui:col': {
+                            md: 12,
+                            children: ['PeriodCycleDurationUnit']
+                          }
+                        }
+                      ]
+                    },
+                    {
+                      'ui:row': [
+                        {
+                          'ui:col': {
+                            md: 12,
+                            children: ['PeriodCycleDurationValue']
+                          }
+                        }
+                      ]
+                    }
+                  ]
+                }
+              }
+            ]
+          }
+        }
+      }
+
+    }
+  },
+  TemporalKeywords: {
+    'ui:heading-level': 'h4',
+    items: {
+      'ui:widget': CustomSelectWidget,
+      'ui:options': {
+        enumOptions: temporalKeywords
+      }
+    }
+
+  },
+  PaleoTemporalCoverages: {
+    'ui:heading-level': 'h4',
+    items: {
+      'ui:field': 'layout',
+      'ui:layout_grid': {
+        'ui:row': [
+          {
+            'ui:col': {
+              md: 12,
+              children: [
+                {
+                  'ui:row': [
+                    {
+                      'ui:col': {
+                        md: 6,
+                        children: ['StartDate']
+                      }
+                    },
+                    {
+                      'ui:col': {
+                        md: 6,
+                        children: ['EndDate']
+                      }
+                    }
+                  ]
+                },
+                {
+                  'ui:row': [
+                    {
+                      'ui:col': {
+                        md: 12,
+                        children: ['ChronostratigraphicUnits']
+                      }
+                    }
+                  ]
+                }
+              ]
+            }
+          }
+        ]
+      },
+      ChronostratigraphicUnits: {
```

Comment:
It looks like the order of these is Eon, Era, Period, Epoch, Stage, Detailed Classification